src/dcTMD/utils/plotting.py

```diff
@@ -134,13 +134,13 @@ def plot_worknormalitychecks(x, workset, index, colors=None):
     plot_worklines(x, workset, axs[0])
 
     if not colors:
-        from matplotlib.cm import get_cmap
-        cmap = get_cmap('Dark2')
+        import matplotlib as mpl
+        cmap = mpl.colormaps['Dark2']
```

Comment:
```suggestion
        cmap = plt.get_cmap('Dark2')
```

Using the novel syntax `mpl.colormaps` it requires `mpl>=3.5.0`. I would rather use the pyplot wrapper to ensure working with all mpl versions.

Reply:
Using plt prevents also to import mpl within the local scope.

Reply:
Thanks, I changed it back.